root/static/scripts/edit/components/Tooltip.js

```diff
@@ -13,28 +13,38 @@
  */
 
 import * as React from 'react';
-import ReactDOM from 'react-dom';
 
 type TooltipProps = {
   +hoverCallback: (bool) => void,
   +content: React.Node,
 };
 
 class Tooltip extends React.Component<TooltipProps> {
+  constructor(props: TooltipProps) {
+    super(props);
+
+    this.containerRef = React.createRef();
+  }
+
   componentDidMount() {
-    const element: any = ReactDOM.findDOMNode(this);
-    const links = element.getElementsByTagName('a');
-    for (let i = 0; i < links.length; i++) {
-      links[i].setAttribute('target', '_blank');
+    const container = this.containerRef.current;
+    const links = container?.getElementsByTagName('a');
+    if (links) {
+      for (let i = 0; i < links.length; i++) {
+        links[i].setAttribute('target', '_blank');
+      }
     }
   }
 
+  containerRef: {current: HTMLDivElement | null};
```

Comment:
Is there a way to mark this as being a type? I'm just used to having types defined first and their usage later, so it confused me for a bit - not a big deal at all, but still.

Reply:
I think eslint required it to be down here because of the [react/sort-comp](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/sort-comp.md) rule. Like you said, it actually makes more sense to me to have all type definitions at the start of the class, so we should probably fix the rule. Would changing the rule and moving it to the top of the class help, or did you have something else in mind?

Reply:
I think that'd make it more clear to me, yes - @yvanzo: what do you think about this case?

Reply:
Just do it! :athletic_shoe:

Reply:
Fixed in the latest commit. :)